scw/custom_types.go

```diff
@@ -164,3 +166,36 @@ func (tsp *TimeSeriesPoint) UnmarshalJSON(b []byte) error {
 
 	return nil
 }
+
+// IPNet inherits net.IPNet and represents an IP network.
+type IPNet struct {
```

Comment:
should it be a type alias ?

Reply:
This way the type inherits `net.IPNet` methods